generator/src/main/java/org/corfudb/generator/distributions/Keys.java

```diff
@@ -0,0 +1,35 @@
+package org.corfudb.generator.distributions;
+
+import java.util.ArrayList;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.UUID;
+
+/**
+ * This class implements the distribution of keys that can be inserted
+ * into a map.
+ *
+ * Created by maithem on 7/14/17.
+ */
+public class Keys extends DataSet {
+    final private Set<String> keys;
+    final private int numKeys;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AModifiersOrderCheck)